package-lock.json

```diff
@@ -6560,28 +6560,27 @@
       "dependencies": {
         "abbrev": {
           "version": "1.1.1",
-          "resolved": false,
+          "resolved": "",
```

Comment:
Why are there changes to package-lock.json?

Reply:
This happens ALL THE TIME... I accidentally checked it in... ugh...

Reply:
Removed these changes.